code/modules/surgery/organs/subtypes/unathi.dm

```diff
@@ -28,3 +28,6 @@
 /obj/item/organ/internal/kidneys/unathi
 	name = "unathi kidneys"
 	icon = 'icons/obj/species_organs/unathi.dmi'
+
+/obj/item/organ/internal/lungs/unathi/ash_walker
+	safe_oxygen_min = 0 // can breathe on lavaland
```

Comment:
This makes them able to breathe everywhere (effectively in vacuum too, only pressure kills them) as they have no `safe_(anygas)_min` value above `0`. I know they had no breath before, but if they were to get real organs, they might be breathing for real as well!

I suggest merely lowering the minimum safe oxygen level, to represent that they, indeed, have a variant of unathi lungs. A much stronger one?

I suggest setting this to `safe_oxygen_min = 8`. It is enough on Lavaland:

![image](https://user-images.githubusercontent.com/33333517/154960607-b8ca9a95-8a00-44bb-b319-abe951db41e5.png)

Lavaland gas/pressure for reference:
```
Pressure: 36.9 kPa
Nitrogen: 62 %
Oxygen: 37 %
CO2: 0 %
Temperature: 26 °C
```

Reply:
Well currently they can breathe without air and as an effort to keep as much functionality the same while fixing the bug i decided zero was an appropriate value. 

If a maint requests this I can change it.